src/nfs.c

```diff
@@ -596,6 +612,34 @@ static int nfs_read(void) {
 }
   /* #endif KERNEL_LINUX */
 
+#elif KERNEL_NETBSD
+static int nfs_read(void) {
+  struct nfsstats ns;
+  size_t size = sizeof(ns);
+  int mib[] = {CTL_VFS, 2, NFS_NFSSTATS};
+  value_t values[nfs3_procedures_names_num];
+  int i;
+
+  /* NetBSD reports v2 statistics mapped to v3 and doen't yet support v4 */
```

Comment:
Maybe add a WARNING or INFO if user tries to config only v2 or v4. For now it will silently omit the parameters and leave the user without any information. This comment is probably the only place where this information is provided.

Reply:
Warnings added.